plugins/database/mysql/mysql.go

```diff
@@ -85,6 +81,40 @@ func (m *MySQL) Initialize(ctx context.Context, req dbplugin.InitializeRequest)
 	if err != nil {
 		return dbplugin.InitializeResponse{}, err
 	}
+
+	usernameTemplate, err := getString(req.Config, "username_template")
```

Comment:
You can probably switch to use `strutil.GetString` from your other PR now that exists.

Reply:
👍 that was the plan!